runtime/compiler/env/VMJ9.cpp

```diff
@@ -8337,6 +8338,14 @@ TR_J9VM::getClassFromMethodBlock(TR_OpaqueMethodBlock *mb)
 TR_IProfiler *
 TR_J9VM::getIProfiler()
    {
+   if (auto stream = TR::CompilationInfo::getStream())
```

Comment:
I would prefer to override TR_J9VM implementation. Is there a reason that is not possible?